docs/get-started/getting-started-alluxio.md

```diff
@@ -0,0 +1,200 @@
+
+   Add Alluxio worker hostnames into `${ALLUXIO_HOME}/conf/workers`.
+
+   ``` json
+   NM_hostname_1
+   NM_hostname_2
+   ```
+
+   - Copy configuration from Alluxio master to Alluxio workers.
+
+   ``` shell
+   ${ALLUXIO_HOME}/bin/alluxio copyDir ${ALLUXIO_HOME}/conf
+   ```
+   - Alluxio worker configuration.
+
+   Add worker and user hostname respectively in the Alluxio configuration of each Alluxio worker.
+
+   ``` xml
+   alluxio.worker.hostname=NM_hostname_X
+   alluxio.user.hostname=NM_hostname_X
+   ```
+
+   Note that Alluxio can manage other storage media (e.g. MEM, HDD) in addition to ssd,
+   so local data access speed may vary depending on the local storage media. To learn
+   more about this topic, please refer to the [tiered storage document](https://docs.alluxio.io/os/user/stable/en/core-services/Caching.html#multiple-tier-storage).
+
+5. Mount an existing S3 bucket to Alluxio.
+
+   ``` bash
+   ${ALLUXIO_HOME}/bin/alluxio fs mount \
+      --option aws.accessKeyId=<AWS_ACCESS_KEY_ID> \
+      --option aws.secretKey=<AWS_SECRET_KEY_ID> \
+      alluxio://RM_hostname:19998/s3 s3a://<S3_BUCKET>/<S3_DIRECTORY>
+   ```
+
+   For other filesystem, please refer to [this site](https://www.alluxio.io/)
+
+6. Start Alluxio cluster.
+
+   Login Alluxio master node, and run
+
+   ``` bash
+   ${ALLUXIO_HOME}/bin/alluxio-start.sh all
+   ```
+
+## RAPIDS Configuration
+
+There are two ways to leverage Alluxio in RAPIDS.
+
+1. Explicitly specify alluxio path.
+
+   This may require user to change code.
+
+   Eg. Change below code
+
+   ``` scala
+   val df = spark.read.parquet("s3a://<S3_BUCKET>/<S3_DIRECTORY>/foo.parquet")
+   ```
+
+   to
+
+   ``` scala
+   val df = spark.read.parquet("alluxio://RM_hostname:19998/s3/foo.parquet")
+   ```
+
+2. Transparently replace in RAPIDS.
+
+   RAPIDS has added a configuration `spark.rapids.alluxio.pathsToReplace` which can allow RAPIDS
+   to replace the input file paths to alluxio paths transparently at runtime. So there is no any
+   code change for users.
+
+   Eg, at startup
+   ``` shell
+   --conf spark.rapids.alluxio.pathsToReplace="s3:/foo->alluxio://RM_hostname:19998/foo,gs:/bar->alluxio://RM_hostname:19998/bar"
+   ```
+
+   This configuration allows RAPIDS to replace any file paths prefixed `s3:/foo` to
+   `alluxio://RM_hostname:19998/foo` and `gs:/bar` to `alluxio://RM_hostname:19998/bar`
+
+3. Submit an application.
+
+   Spark driver and tasks will parse `alluxio://` schema and access Alluxio cluster by
+   `alluxio-${LATEST}-client.jar` which must be distributed across the all nodes where Spark drivers
+   or executors are running.
+
+   The Alluxio client jar must be in the classpath of all Spark drivers and executors in order for
+   Spark applications to access Alluxio.
+
+   We can specify it in the configuration of `spark.driver.extraClassPath` and
```

Comment:
you shouldn't need to put it in the extraClasspath, just specify it with the --jars option on submission.

Reply:
that's is weird, what exactly is error?  Is it on driver or executors? maybe its to early on startup, if this is on yarn it should distribute it and be in class path though before process starts

Reply:
if specifying in extraClassPath that means it has to be present on the nodes. I can see this possibly for the driver node needing, but on yarn it shouldn't be needed for the executors. the --jar option on yarn copies the jar to the node and it should be in class path when the executor starts.  I'm ok with leaving it as is but we should state it has to be on nodes

Reply:
Done